frontend/public/components/utils/timestamp.jsx

```diff
@@ -75,10 +75,14 @@ export class Timestamp extends SafetyFirst {
     }
 
     const timeAgo = now - mdate;
+    if (this.props.omitSuffix) {
```

Comment:
Prob should test, above you have `noSuffix` rather than `omitSuffix`:  `export const fromNow = (dateTime, now=undefined, noSuffix=false) => {`